javascript/browser-apis/dom-manipulation/write-a-function-to-return-a-complex-element.md

```diff
@@ -66,16 +55,17 @@ function createListElement(data){
   return ul;
 }
 
-var teaList = createListElement(
+let teaList = createListElement(
   teas
 );
+
 container.appendChild(
   teaList
 );
 
 ```
 
-In the example above we selected a parent element and stored it in a variable called container. Then we created a ul element, looped through our array of teas and created a li element for each type of tea while appending each of them to the ul. Once we created the list we appended it to the parent element.
+In the example above we selected a parent element and stored it in a variable called `container`. Then we created an `ul` element, looped through our array of `teas` and created a `li` element for each type of `tea` while appending each of them to the `ul`. Once we created the list we appended it to the parent element.
```

Comment:
I think in this case it should be **a `ul`**, because **u** is read as **yu**
```suggestion
In the example above we selected a parent element and stored it in a variable called `container`. Then we created a `ul` element, looped through our array of `teas` and created a `li` element for each type of `tea` while appending each of them to the `ul`. Once we created the list we appended it to the parent element.
```